### ISSUE TITLE
Editing the existing comment makes it harder to find

### ISSUE THREAD

**Issue by alexeagle** [Daniel15/BuildSize#33]
When I publish a new commit to a PR, buildsize updates the comment it made earlier.

This is nice to reduce spaminess on the thread, but on the other hand it means the comment gets lost higher up the thread, eg. https://github.com/angular/angular/pull/22040 has some later robot comments added for each commit.

Not sure if there's a way to have both low-spam and also keep the message near the bottom. Maybe it's a moot point if we solve #31 since that way the PR status will indicate the current state of the PR.

**Comment by alexeagle** [comment]
For reference (in case that PR changes) here's what it looks like now
![rppe6slfkgb](https://user-images.githubusercontent.com/47395/35873798-a3222e3e-0b1f-11e8-84eb-a709c9c834ce.png)

**Comment by Daniel15** [comment]
Yeah, this is tricky... One option is to delete the old comment then post a new one, but then you lose historical context. Maybe a good idea is to edit the comment if there's just a small change, but post a new comment if the change is significant?
